### PR TITLE
:memo: add readme and makefile

### PR DESCRIPTION
**Pull request by matyo91**
I want to improve tune-manager usability

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
Wow thanks!

I had no idea anyone was using this at all :)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+##
+##usage :
+##-------
```
Comment (EvanPurkhiser) [inline Makefile:3]: Could you remove this comment?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+build: ## build
```
Comment (EvanPurkhiser) [inline Makefile:5]: Not sure if this comment is needed eiter

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	docker build --tag=evanpurkhiser/tune-manager:latest .
+
+dev: build ## dev
+	docker run -it -p 8080:80 -v library:/library -v staging:/staging -v storage:/storage evanpurkhiser/tune-manager
+
+# DEFAULT
+.DEFAULT_GOAL := help
+help:
+	@grep -E '(^[a-zA-Z_-]+:.*?##.*$$)|(^##)' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
```
Comment (EvanPurkhiser) [inline Makefile:14]: What does this do?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Keep your music library organized.
+
+## Run localy with docker
+
+```bash
+$ make dev
+```
+
+Then browse `http://localhost:8080`
```
Comment (EvanPurkhiser) [inline README.md:11]: Might be more worthwhile for it to be `http://localhost:8080/importer`

**Review by EvanPurkhiser — CHANGES_REQUESTED** [review]
Left some change comments

File: Makefile (hunk 3 above)
Comment (matyo91) [reply Makefile:14]: This trick allows a nice looking help when you tip `make`

actually this render

```
usage :                      
-------                      
build                          build
dev                            dev
```

It's compatible withs comments into the `Makefile`. I can delete then if you don't want it.

**Comment by matyo91** [comment]
I did write about it. I think use it in the futur to manage my music library.

https://blog.darkwood.fr/article/organiser-son-dossier-de-musique-pour-dj-sur-le-cloud

When i get time, i wish make interfaces about choosing : paths naming, pcloud file managment.

**Comment by matyo91** [comment]
I'm not really using it now. I hope having an integration with music [brain picard](https://picard.musicbrainz.org) too !

File: Makefile (hunk 2 above)
Comment (matyo91) [reply Makefile:5]: see https://github.com/EvanPurkhiser/tune-manager/pull/13#discussion_r520578195

File: Makefile (hunk 1 above)
Comment (matyo91) [reply Makefile:3]: see https://github.com/EvanPurkhiser/tune-manager/pull/13#discussion_r520578195

**Comment by EvanPurkhiser** [comment]
I've been working on more stuff on this recently too actually :)
image

**Comment by matyo91** [comment]
finally I could run it locally ! 🎉

**Comment by EvanPurkhiser** [comment]
Good to hear!

I realize this project is in a very "not done" state, it's definitely one of my more "private" public repos, but if you want to give it a shot, the basic premise is that you have a 'Library' and a 'imports folder'. The library is your collection of music, the imports folder is 'new tracks that go into my library'.

I've never done "big" imports before, but you would likely want to put your collection into the 'imports' folder to setup the library the way it likes.

The `/importer/` stuff is how I import new music into my collection. It does key analysis, and imports metadata for beatport tracks. Then it will also tell you if the artists and labels already exists and ensure's the casings are correct

**Comment by EvanPurkhiser** [comment]
I'll take a closer look at this PR again sometime soon here, just busy with other things right now as well :)

**Comment by matyo91** [comment]
Okey !

Indeed I didn't made mass import of my music library ! Before I do that, I want to be comfortable using tune-manager to not loose any music track.
To run it locally I need upgrade pillow to ^8.0 because of fail compilation on MacOS BigSur.

The way I think using it is :
- When I get a new tracks, I use Music Brain to mass tag tracks.
- Then I put all new tracks into 'staging' folder (I may make a PR to rename the folder into 'import')
- Then import tracks by using the web interface http://localhost:8080/importer
- In future PR, I may add a new File connector to manage 'staging' and 'library' folder directly in pCloud.

**Pull request closed without merging**